torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -898,6 +927,49 @@ def _run_param_init_fn():
         # For validating execution order across ranks
         self._exec_order_data = _ExecOrderData()
 
+    def _init_param_exec_order_wrap_policy(
+        self,
```

Comment:
Since this is just forwarding the constructor arguments, could we replace this with `*args, **kwargs` and simply forward them to `__init__()`?

Otherwise, every time that we add a new argument to the `FullyShardedDataParallel` constructor, we need to modify this method's signature as well.

Reply:
Thanks! Done